front/pages/w/[wId]/data-sources/vaults/[vaultId]/categories/[category]/index.tsx

```diff
@@ -77,6 +121,39 @@ export default function Vault({
   systemVault,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
   const router = useRouter();
+  const sendNotification = useContext(SendNotificationsContext);
+
+  const handleSelect = async (sId: string) => {
+    switch (category) {
+      case "apps": {
+        await router.push(`/w/${owner.sId}/a/${sId}`);
+        break;
+      }
+      case "website": {
+        // for websites we want to skip the root node and directly redirect to
+        // the next depth level
+        const url = await fetchWebsiteRootUrl(owner, vault, category, sId);
+        if (url) {
+          await router.push(url);
+        } else {
+          sendNotification({
```

Comment:
Instead of that we could simply fall back on the default case instead? 
(i.e. redirect to `/w/${owner.sId}/data-sources/vaults/${vault.sId}/categories/${category}/data_source_views/${sId}`) ?